store/tikv/kv.go

```diff
@@ -241,7 +241,7 @@ func (s *tikvStore) runSafePointChecker() {
 				d = gcSafePointUpdateInterval
 			} else {
 				loadSafepointCounter.WithLabelValues("fail").Inc()
-				log.Errorf("[tikv] fail to load safepoint: %v", err)
+				log.Errorf("[check visibility] fail to load safepoint from pd: %v", err)
```

Comment:
Maybe this gives more information?
[tikv] check visibility fail to ...

Reply:
The message comes from pd, not tikv, so I remove the flag. I think `check visibility` is useless either.^^